hydrofunctions/hydrofunctions.py

```diff
@@ -584,8 +584,8 @@ def extract_nwis_df(nwis_dict, interpolate=True):
     zero = to_offset("0min")
     freqs2 = list(filter(lambda x: x > zero, freqs))
     if len(freqs2) > 0:
-        freqmin = min(freqs)
-        freqmax = max(freqs)
+        freqmin = min(freqs2)
```

Comment:
This is great- it sets the freqmin to the smallest frequency in the list that is larger than zero.
I should probably change 'freqs2' to a more meaningful name, like 'freqs_no_zeroes'.